packages/core/src/grid/docs/all.story.ts

```diff
@@ -38,6 +38,28 @@ import { rowSwappable } from './row-swappable.story.js';
 import { rtl } from './rtl.story.js';
 import { noScroll } from './scroll-height.story.js';
 
+import { registerElementSafely } from '@cds/core/internal';
+import {
+  CdsGrid,
+  CdsGridRow,
+  CdsGridCell,
+  CdsGridColumn,
+  CdsGridDetail,
+  CdsGridFooter,
+  CdsGridPagination,
+  CdsGridPlaceholder,
+} from '@cds/core/grid';
+import '@cds/core/grid/register.js';
+
+registerElementSafely('cds-grid', CdsGrid);
```

Comment:
We're only doing this here now because it was commented out in `packages/core/src/grid/register.ts`, right?

Reply:
Yeah, to release the 6.x stable I disabled the registration in the published package. This adds the registration to storybook so locally in development we still have access. I'll add a comment to this and the other register file with more detail

Reply:
I'm going to rework this approach. Disabling registration outright causes cascading failures to all the other package unit tests as well.

Reply:
Agree, I'm going to leave them alone as mark the components as `@internal` rather than `@beta`.

Reply:
migrated to `@internal`